src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_agentpool_decorator.py

```diff
@@ -1267,6 +1267,26 @@ def common_get_no_wait(self):
         )
         self.assertEqual(ctx_1.get_no_wait(), False)
 
+    def common_set_up_gpu_propertes(self):
```

Comment:
missing several tests like [common_get_gpu_instance_profile](https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/tests/latest/test_agentpool_decorator.py#L169), [standalone_test_get_gpu_instance_profile](https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/tests/latest/test_agentpool_decorator.py#L283), [mc_test_get_gpu_instance_profile](https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/tests/latest/test_agentpool_decorator.py#L318), [standalone_test_set_up_gpu_propertes](https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/tests/latest/test_agentpool_decorator.py#L471), [mc_test_set_up_gpu_propertes](https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/tests/latest/test_agentpool_decorator.py#L563)

Reply:
oops, will add

Reply:
think I got them all?

Reply:
Yes, but this function seems to be called nowhere, maybe remove it?

Queued a [pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=58959702&view=results) to validate the live test case.

Reply:
seems good?

Reply:
Oops, sorry for the late reply. Looks all good to me :shipit: